modules/shared/src/main/scala/io/constellationnetwork/env/JarSignature.scala

```diff
@@ -0,0 +1,31 @@
+package io.constellationnetwork.env
+
+import java.security.MessageDigest
+
+import cats.effect.IO
+
+import io.constellationnetwork.security.hash.Hash
+import io.constellationnetwork.security.hex.Hex
+
+import fs2.io.file.{Files, Path}
+
+object JarSignature {
+
+  val jarHash: IO[Hash] = {
```

Comment:
please use tagless algebra that we use in the whole codebase. define the least type required by the function and do not use the `IO` type here. `IO` is the type provided when you run the code, so in the `TessellationIOApp` or tests